### ISSUE TITLE
Could not open configuration file Error:  open /etc/C:\Users\

### ISSUE THREAD

**Issue by finkr** [cblomart/vsphere-graphite#85]
Hello,

I tried vsphere-graphite on Windows. I had to recompile it because it search the configuration file in `/etc/${binaryname}.json`.

Regards,

Franklin

**Comment by cblomart** [comment]
So i have the following error

```log
2019/04/02 11:20:54 Could not open configuration file Error:  open /etc/vsphere-graphite.exe.json: The system cannot find the path specified.
```

I will look at detecting file presence to look in two places:
1/ in "/etc/{{binaryname}}.json"
2/ in "./{{binaryname}}.json" (current folder)

**Comment by cblomart** [comment]
Can you check that this work better with v0.7c? 
The config file should be in the executable directory.

**Comment by finkr** [comment]
I'll try it soon.

Thank you

**Comment by cblomart** [comment]
Thanks @finkr for your help identifying and correcting this bug.
I re-released the version v0.7c.

PS: i know, i am not happy with re-releasing... currently the ci/cd pipeline doesn't make a windows binary available for each build ... i wanted it from the ci/cd to test.

**Comment by cblomart** [comment]
I will close the issue. 
Naturally don't hesitate to open it up again if there are still some issues.
